sdk/azurestackhci/Microsoft.Azure.Management.AzureStackHCI/src/Generated/ClustersOperations.cs

```diff
@@ -74,7 +74,7 @@ internal ClustersOperations(AzureStackHCIClient client)
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<IPage<Cluster>>> ListWithHttpMessagesAsync(Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
This is definitely a breaking change since we are changing the method signature.  Unfortunately this doesn't seem to have been caught in the spec pr https://github.com/Azure/azure-rest-api-specs/pull/10953.  Can you please work with the breaking change approval board to get the Approved-BreakingChange label added to the spec PR?

Reply:
@m-nash The swagger PR has the Approved-BreakingChange label now.

Reply:
I see the label got added, closing this out now